JamGame/Assets/Scripts/Utils/ProjectedTilesValidator.cs

```diff
@@ -0,0 +1,39 @@
+﻿#if UNITY_EDITOR
+using System.IO;
+using TileUnion;
+using UnityEditor;
+using UnityEngine;
+
+namespace Utils
+{
+    internal class ProjectedTilesValidator
+    {
+        private static int projectedCount = 10;
```

Comment:
Please introduce a config for this or just open a dialog window where you can select amount when menu item is selected